group_vars/all.yml

```diff
@@ -39,9 +39,9 @@ machine_users:
 galaxy_user:
   name: galaxy
   shell: /bin/bash
-  uid: 998
+  uid: 1024
```

Comment:
If we have uid: "{{ galaxy_user_id }}"  we can be defining this in new group_vars files for pawsey, dev, staging

Reply:
yeah, good idea. I'll get onto it after I've stood up the workers and the queue server